.ci/gcb-changelog-checker.yml

```diff
@@ -0,0 +1,17 @@
+---
+steps:
+- name: 'hashicorpdev/go-changelog'
+  id: check-changelog
+  entrypoint: /go-changelog/changelog-pr-body-check
```

Comment:
I worry about making more cloud build ops because I think there is a limit to how many cb runners can execute on a project simultaneously

Reply:
Good to know! It looks like the changelog checker takes ~2 seconds so I think it will not be too disruptive.